src/main/scala/codacy/plugins/DockerTest.scala

```diff
@@ -13,7 +13,7 @@ case class Sources(mainSourcePath: Path, directoryPaths: Seq[Path])
 object DockerTest {
 
   private val allTestKey = Seq("all")
-  private val possibleTests = Seq(JsonTests, PluginsTests, PatternTests)
+  private val possibleTests = Seq(JsonTests, PluginsTests, PatternTests, SourceTests)
```

Comment:
We cannot add this here.

Since we have most tools running with flag `all` we need to create a new `allWithSources`